docs/docker-compose/start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+git clone https://github.com/ceph/paddles.git
+cd paddles
+git fetch origin pull/94/head:wip-amathuria-removing-beanstalkd
+git checkout wip-amathuria-removing-beanstalkd
+cd ../
+git clone https://github.com/ceph/pulpito.git
+git clone https://github.com/ceph/teuthology.git
+cd teuthology
```

Comment:
What's the purpose of cloning another copy of teuthology?

Reply:
Needed to use #1650 version of technology to run dispatcher without beanstalk.